lib/libutils/isoc/include/setjmp.h

```diff
@@ -56,7 +56,6 @@
 typedef	_JBTYPE jmp_buf[_JBLEN];
 #endif
 
-void __noreturn longjmp(jmp_buf env, int val);
```

Comment:
We need longjump for xtest like test_setjmp. Moreover, this is common header file.